hazelcast-oss/Dockerfile

```diff
@@ -69,6 +73,7 @@ CMD ["bash", "-c", "set -euo pipefail \
       && if [[ \"x${JAVA_OPTS}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS_DEFAULT} ${JAVA_OPTS}\"; else export JAVA_OPTS=\"${JAVA_OPTS_DEFAULT}\"; fi \
       && if [[ \"x${MIN_HEAP_SIZE}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Xms${MIN_HEAP_SIZE}\"; fi \
       && if [[ \"x${MAX_HEAP_SIZE}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Xmx${MAX_HEAP_SIZE}\"; fi \
+      && if [[ \"x${MANCENTER_URL}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Dhazelcast.mancenter.enabled=true -Dhazelcast.mancenter.url=${MANCENTER_URL}\"; fi \
```

Comment:
It should normally be sufficient to enable MC just passing `-Dhazelcast.mancenter.url` as System Property so that we don't have to maintain hazelcast.xml...

It is actually not the only concern. The other one is if hazelcast.xml is provided by user, she sets MANCENTER_URL but forgets about adding those system properties into hazelcast.xml, then you end up with incorrect MC configuration. cc: @emre-aydin

Reply:
Should be easy to add those as group properties to members, but AFAIK introducing new group properties are generally frowned upon by the core team. WDYT @mmedenjak ?

Reply:
I don't like group properties but sometimes (rarely) I would allow it. On the other hand, you **DO NOT** want to mention adding group properties to Greg :)
Please refer to: https://hazelcast.atlassian.net/wiki/spaces/EN/pages/253953578/Hazelcast+Configuration

Reply:
I think we can keep what we have now but please keep in mind enabling MC is different than configuring hazelcast product. We might need to have an switch to on/off MC in cloud environments without touching hazelcast.xml. OTOH, if MC team implements auto discovery of hazelcast cluster, we will even not need any config on hazelcast side. It is best to wait for auto-discovery then. cc: @emre-aydin